### PR TITLE
Update readme with more package info

### PR DESCRIPTION
**Pull request by mattdangerw**
Not sure about the timeline bit? We can just axe that if preferred.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,25 +5,46 @@
...
+## Installation
+
+### Install with pip
+
+Keras 3 is available as a preview release on PyPI named `keras-core`.
+Keras 2 (`tf.keras`) is distributed along with the `tensorflow` package.
+
+1. Install `keras-core`:
+
+    ```
```
Comment (fchollet) [inline README.md:17]: Unindent code block

File: README.md [hunk 2]
```diff
@@ -5,25 +5,46 @@
...
+    pip install keras-core
+    ```
+
+2. Install backend package(s).
+
+    To use keras-core, you should also install the backend of choice: `tensorflow`, `jax`, or `torch`.
```
Comment (fchollet) [inline README.md:23]: Unindent this paragraph as well

File: README.md [hunk 3]
```diff
@@ -5,25 +5,46 @@
...
+### Local installation
 
 Keras 3 is compatible with Linux and MacOS systems. To install a local development version:
 
 1. Install dependencies:
 
-```
-pip install -r requirements.txt
-```
+    ```
```
Comment (fchollet) [inline README.md:33]: Unindent

File: README.md [hunk 4]
```diff
@@ -5,25 +5,46 @@
...
+    pip install -r requirements.txt
+    ```
 
 2. Run installation command from the root directory.
 
-```
-python pip_build.py --install
-```
+    ```
```
Comment (fchollet) [inline README.md:39]: Unindent

File: README.md [hunk 5]
```diff
@@ -5,25 +5,46 @@
...
+    python pip_build.py --install
+    ```
+
+3. Add accelerator support for the backend(s) of your choice.
 
-You should also install your backend of choice: `tensorflow`, `jax`, or `torch`.
-Note that `tensorflow` is required for using certain Keras 3 features: certain preprocessing layers as
-well as `tf.data` pipelines.
+    The `requirements.txt` file will install a CPU-only version of TensorFlow, JAX,
```
Comment (fchollet) [inline README.md:45]: Unindent

**Comment by mattdangerw** [comment]
Removed the indents, though the rendered visual indent on github will get a bit weird with this ->
https://github.com/mattdangerw/keras/tree/readme-updates#installation

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Pull request merged**